lib/Services/Bridge/API.php

```diff
@@ -175,4 +186,13 @@ private function request($url, $body, $method = 'POST') {
     ];
     return $this->wp->wpRemotePost($url, $params);
   }
+
+  private function logCurlError(WP_Error $error) {
+    $logData = [
+      'curl_errno' => $this->curlHandle ? curl_errno($this->curlHandle) : 'n/a',
+      'curl_error' => $this->curlHandle ? curl_error($this->curlHandle) : $error->get_error_message(),
+      'curl_getinfo' => $this->curlHandle ? curl_getinfo($this->curlHandle) : 'n/a',
```

Comment:
I noticed that in `logCurlInformation`, we use the key `curl_info`. I would prefer to use the same key, but it is nothing important. What do you think?